packages/components/psammead-include/src/index.jsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/no-danger */
+import React from 'react';
+import { string } from 'prop-types';
+
+const Include = ({ html }) => {
```

Comment:
As the only thing this component is doing is using `dangerouslySetInnerHTML` with a single prop of HTML should we tie it with the naming convention of Include? I feel like the naming is very specific to something which is internal to the BBC.

Reply:
@andrewscfc will be using this component for his IDT1 work.

Reply:
I will be using it but one thing I would say is much of the heavy lifting is being done in simorgh in a container i.e. fetching the include from the endpoint. That brings us back to the discussion about whether we should have a psammead component at all for this, maybe just 'dangerously set' the html in the container and don't bother with a component?

The story demonstrates some of the value as it takes a chromatic snapshot of a real include successfully loading, it encapsulates the knowledge that embedding an include just involves dangerously setting the html.

Let's see what this looks like with the simorgh PR and have another think?

Reply:
Maybe having it as a component in Simorgh? This component does a very similar thing https://github.com/bbc/simorgh/blob/latest/src/app/components/IfAboveIE9Comment/index.jsx. 

I think you're right though using it in Simorgh PR and thinking about it later